parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -206,6 +206,16 @@ EOF
     fi
 fi
 
+if [[ "${AMD_GPU_NODE}" = true ]] && [[ "${skip_gpu_driver_install}" != "true" ]]; then
+    logs_to_events "AKS.CSE.ensureAMDGPUDrivers" ensureAMDGPUDrivers
+else
```

Comment:
should we have those in a reusable function ? I would expect that we run a cleanup logic after we install the GPU drivers

Reply:
Here I deleted AMD-related logic for non-AMD users.

Probably can cleanup for everyone.